Log command start and finish

kirillgarbar opened the pull request:
To configure logging for each command and group in chadmin custom command group had to be added, like for monrun_checks and monrun_checks_keeper.

MedvedewEM commented on ch_tools/chadmin/cli/chadmin_group.py:
```diff
@@ -0,0 +1,62 @@
+                __version__,
+            )
+
+            result = cmd_callback(*a, **kw)
```
Do we log something in case of error/fail in `cmd_callback`?
Maybe add some common message like in `monrun_checks_keeper` [here](https://github.com/yandex/ch-tools/pull/210/files#diff-073ad2263dfe496edc132c2c57a8d2d146262e3e5da43bbd02d8c0a5b1c4e51aR73)?

aalexfvk replied:
It was wrapped in `try...except`

kirillgarbar commented:
Also fixed messages like `[DEBUG   ] chadmin: started: Executing command 'wait'` instead of `[DEBUG   ] chadmin: started: Executing command 'started'` because we added command groups to another group but with the same `add_command` method
